tests/test_recipe.py

```diff
@@ -125,7 +125,7 @@ def test_dimension2(self):
        sum(foo.age) AS age
 FROM foo
 GROUP BY last
-ORDER BY foo.last"""
+ORDER BY last"""
```

Comment:
This was failing tests. I assumed it's caused by changes in recipe so just updated the expected value, haven't confirmed it though.